### PR TITLE
adding adversary village page

### PR DESCRIPTION
**Pull request by xFreed0m**

### CODE REVIEW

File: content/adversary_village/_index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- Regular talk (25 minutes)
+- Deep-dive, technical talk (45 minutes, which should include a workshop or a live demo)
+- Lightning talk (10 minutes)
+- Panel (30 minutes, which should include a few members)
+
+Submit your proposal here:
+
+#### Call for Volunteers
+
+We're looking for volunteers to help us run the Adversary Village this year. If you're interested in joining our team, please reach out to us via email at ‫adversaryvillage@bsidestlv.com‬ or on Twitter @AdversaryVillageIL.
```
Comment (omercnet) [inline content/adversary_village/_index.md:47]: can you replace the unicode chars?

File: content/adversary_village/_index.md
Comment (omercnet) [inline content/adversary_village/_index.md:1]: need to create corresponding css files for new pages, see styles folder
https://github.com/bsidestlv/bsidestlv.com/tree/master/src/styles

**Review by omercnet — CHANGES_REQUESTED** [review]
see comments

File: content/adversary_village/_index.md (hunk 1 above)
Comment (xFreed0m) [reply content/adversary_village/_index.md:47]: moved it into a link

File: content/adversary_village/_index.md
Comment (xFreed0m) [reply content/adversary_village/_index.md:1]: yes just did it, pushing now

File: content/adversary_village/_index.md (hunk 1 above)
Comment (omercnet) [reply content/adversary_village/_index.md]: image
still seeing it

**Comment by xFreed0m** [comment]
> see comments

fixed now

File: content/adversary_village/_index.md
Comment (xFreed0m) [reply content/adversary_village/_index.md:1]: pushed

**Review by guybarnhartmagen — APPROVED** [review]

**Pull request merged**